docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc

```diff
@@ -511,3 +511,10 @@ installed integrations.
 
 . Under advanced settings, set `windows.advanced.diagnostic.enabled`
 to `false`, and then save the integration.
+
+
+[discrete]
+[[hosted-agent-offline]]
+== Hosted {agent} Offline
+
+In Elastic Cloud on scaling up or down the hosted {agent}, new containers for the Elastic Agent are started and the old ones are shut down. The old Elastic Agents will show up in the list for 24 hours and will automatically disappear afterwards.
```

Comment:
Before I review this, I need to know: Are you talking about the Elastic Agent running Fleet Server? Or an agent that's running to monitor Elastic cloud (I'm assuming there is one)? Or both? I just want to make sure we're clear about what hosted Elastic Agent means.

Reply:
The one that has that "Elastic Agent cloud policy" with fleet-server integrations inside assigned.

![Screenshot 2021-11-30 at 20 22 30](https://user-images.githubusercontent.com/244900/144113761-ef4aba0e-a3d0-4237-8d47-87911e333aea.png)